src/main/java/org/sagebionetworks/bridge/sdk/models/accounts/StudyParticipant.java

```diff
@@ -72,9 +70,7 @@
         this.status = status;
         this.id = id;
         this.languages = languages;
-        if (attributes != null) {
-            this.attributes.putAll(attributes);
-        }
+        this.attributes = attributes;
```

Comment:
I think there's some confusion here. This is super easy to fix: just move the Immutable stuff to the constructor. I don't believe there's any downside to doing this.